WooCommerce/Classes/ViewRelated/Orders/FulfillViewController.swift

```diff
@@ -178,6 +180,18 @@ extension FulfillViewController {
 
         AppDelegate.shared.noticePresenter.enqueue(notice: notice)
     }
+
+    /// Displays the `Unable to Fulfill Order` Notice.
+    ///
+    func displayErrorNotice(orderID: Int) {
+        let title = NSLocalizedString("Unable to fulfill order #\(orderID)", comment: "Content of error presented when Fullfill Order Action Failed")
```

Comment:
@ctarda whenever we are adding vars localized string, we (typically) try to help out the translators as much as possible in the comment by adding something like `comment: "Content of error presented when Fullfill Order Action Failed. It reads: Unable to fulfill order #{order number}"`. This helps them understand the full context and they can then move the substitution value around if needed.